### PR TITLE
add libvirt-devel package to tooling images

### PR DESCRIPTION
**Pull request by paulfantom**
Add libvirt-devel package needed by cluster-api-provider-libvirt

/CC: @stevekuznetsov

### CODE REVIEW

**Comment by wking** [comment]
CC @rabugopsl.  I dunno if you'll need these to libvirt smoke tests or not.  If we need to run the libvirt smoke tests on AWS, probably not.  But if we're allowed to run those on the Prow boxes, this might be helpful.

File: projects/origin-release/golang-1.10/Dockerfile [hunk 1]
```diff
@@ -25,7 +25,7 @@ RUN yum install -y epel-release && \
     git config --system user.email origin-release@redhat.com
 
 ADD *.rpm /tmp/origin-rpm/
-RUN yum install -y /tmp/origin-rpm/*.rpm && \
+RUN yum install -y /tmp/origin-rpm/*.rpm libvirt-devel && \
```
Comment (petr-muller) [inline projects/origin-release/golang-1.10/Dockerfile]: Hm, is there a reason why you haven't added the package to the big `INSTALL_PKGS` list ten lines above this?

**Comment by petr-muller** [comment]
Otherwise this looks fine to me, but I would like someone with more knowledge about what this can actually cause to approve.

**Comment by paulfantom** [comment]
Closes #1415

**Comment by stevekuznetsov** [comment]
Are you going to try to run `libvirt` inside of a container in a k8s pod? Does that even work?

**Comment by paulfantom** [comment]
No, we need libvirt libraries to compile/test/lint cluster-api-provider-libvirt. We don't need libvirt, just it's dev libs.

**Comment by paulfantom** [comment]
However now I am curious if it is possible to run libvirt in k8s pod... :smile:

**Comment by paulfantom** [comment]
@petr-muller @stevekuznetsov can those images be rebuilt and pushed to docker hub? Last push was 3 days ago and it doesn't contain changes from this PR.

**Comment by stevekuznetsov** [comment]
I triggered builds, they should finish in a couple minutes

**Comment by paulfantom** [comment]
Thanks! :+1:

**Pull request merged**